STX.REST.RESTFulSense.Clients.Tests.Unit/Services/Foundations/HttpExchanges/HttpExchangeServiceTests.cs

```diff
@@ -255,7 +255,9 @@ private static dynamic CreateRandomHttpContent()
         private static dynamic CreateRandomHttpProperties(
             bool sendUrlParameters,
             bool sendRequestHeaders,
-            bool sendRequestContent)
+            bool sendRequestContent,
+            bool sendRequestContentHeaders,
```

Comment:
why sendRequestContentHeaders?, sending a content implies send the headers

Reply:
got you. resolved